Dockerfile

```diff
@@ -208,6 +208,18 @@ RUN mkdir -p "$RUSTUP_HOME" && chown dependabot:dependabot "$RUSTUP_HOME"
 USER dependabot
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y \
   && rustup toolchain install 1.51.0 && rustup default 1.51.0
+
+
+### Terraform
+
+USER root
```

Comment:
Can we run this as `USER dependabot`? We might run into issues using the binary otherwise?

Reply:
I think root is required for `apt-` stuff, should be fine as long as we don't need to modify it

Reply:
Looks like root is required for both `apt-key add` and `apt-get`

![Screen Shot 2021-06-07 at 11 30 07 AM](https://user-images.githubusercontent.com/12107187/121049912-27127c00-c786-11eb-8a77-1c705def7154.png)

![Screen Shot 2021-06-07 at 11 47 48 AM](https://user-images.githubusercontent.com/12107187/121049949-309be400-c786-11eb-97c0-45c0c5f9cfec.png)